packages-content-model/roosterjs-content-model-editor/lib/publicTypes/enum/BorderOperations.ts

```diff
@@ -5,4 +5,32 @@ export type BorderOperations =
     /**
      * Apply border format to all borders
      */
-    'AllBorders';
+    | 'AllBorders'
```

Comment:
Not a big problem. All existing string literal values are started with lower case. There is no strong reason but I still want to make them consistent, either all lower case or all upper case.

Reply:
You mean for example changing AllBorders to allBorders?

Reply:
Yes.